hs_core/templates/search/search.html

```diff
@@ -999,13 +1016,13 @@ <h4 class="panel-title">
             var legend_content = [];
             var geocoder_content = [];
             var resetButton_content = [];
-            legend_content.push("<p><img src='//cdn.rawgit.com/Concept211/Google-Maps-Markers/master/images/marker_red.png'> Point Coverage Resources </p>");
-            legend_content.push("<p><img src='//cdn.rawgit.com/Concept211/Google-Maps-Markers/master/images/marker_blue.png'> Box Coverage Resources </p>");
-            legend_content.push("<p><img src='//cdn.rawgit.com/googlemaps/js-marker-clusterer/gh-pages/images/m2.png'> number of sites </p>");
+            legend_content.push("<p><img src='//cdn.rawgit.com/Concept211/Google-Maps-Markers/master/images/marker_red.png'> Point Coverage Locations </p>");
+            legend_content.push("<p><img src='//cdn.rawgit.com/Concept211/Google-Maps-Markers/master/images/marker_blue.png'> Box Coverage Centers </p>");
+            legend_content.push("<p><img src='//cdn.rawgit.com/googlemaps/js-marker-clusterer/gh-pages/images/m2.png'> Clusters </p>");
```

Comment:
The layout on this information could be improved. I suggest using an html table with two columns, one for the legend icon and the other for the description. The icons can be smaller if they use too much space.